library/src/main/java/bio/terra/pfb/PfbReader.java

```diff
@@ -97,6 +93,14 @@ public static Metadata getPfbMetadata(String fileLocation) throws IOException {
     throw new InvalidPfbException("Error reading PFB Metadata object");
   }
 
+  // note that this does not decode enum values via convertEnum. WDS does not need decoding;
+  // if other clients do need it we should add it in here.
+  public static List<Schema> getPfbSchema(String fileLocation) throws IOException {
+    return readPfbSchema(fileLocation).getField("object").schema().getTypes().stream()
+        .filter(t -> !t.getName().equals("Metadata"))
```

Comment:
I see a couple hardcoded strings in here which makes me squint a bit more...is there really a thing called `Metadata` that is exact-cased like that?  Should we make the comparison ignore case?

Reply:
These are PFB-specific names. In short, the PFB format is an Avro file where:
* the top-level schema has three fields: "id", "name", and "object"
* the first row will have id=null, name="Metadata", and object={the pfb metadata}
* second and following rows will have id=non-null, name=non-null, and object={attributes for this row}